prow/scripts/compatibility-cli.sh

```diff
@@ -17,24 +17,18 @@ VARIABLES=(
     COMPAT_BACKTRACK
 )
 
-for var in "${VARIABLES[@]}"; do
-    if [ -z "${!var}" ] ; then
-        echo "ERROR: $var is not set"
-        discoverUnsetVar=true
-    fi
-done
-if [ "${discoverUnsetVar}" = true ] ; then
-    exit 1
-fi
-
 #Exported variables
 export KYMA_PROJECT_DIR=${KYMA_PROJECT_DIR:-"/home/prow/go/src/github.com/kyma-project"}
 export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
 export TEST_INFRA_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts"
 
+# shellcheck disable=SC1090
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/testing-helpers.sh"
 # shellcheck disable=SC1090
 source "${TEST_INFRA_SCRIPTS}/library.sh"
 
+checkRequiredVars "${VARIABLES[@]}"
+
```

Comment:
I cannot see the `utils.sh` file sources there and also it seems that the function name is wrong. it should be `utils::check_required_vars`

Reply:
Fixed source from `lib/testing-helpers.sh` to `lib/utils.sh`